### DISCUSSION TITLE
小哥有开交流群吗？

### DISCUSSION THREAD

**Discussion by sansan-mei** [win12-online/win12#205] [Q&A] [answered]
微信、qq、discord等等都行

**Comment by User782Tec** [comment, marked as answer]
目前没有qwq
目前可以用Github的Discussions交流

**Comment by tjy-gitnub** [comment]
你要加交流群干嘛？

**Comment by lingbopro** [comment]
> 目前没有qwq
> 目前可以用Github的Discussions交流

不是有Teams吗？

**Comment by Mbilse** [comment]
> 微信、qq、discord等等都行

我有企业微信,teams和discord群
等等，没人用企微政务版？
